src/components/InputMediaGroup/MediaPreview/MediaPreview.types.ts

```diff
@@ -1,8 +1,8 @@
-import { IMedia } from '~components/InputMedia/InputMedia.types';
+import { IMedia } from '../InputMedia/InputMedia.types';
 
-export interface IMediaGroupProps {
+export type IMediaPreviewProps = {
```

Comment:
Tá parecendo algo que o lint/prettier mudou, ai é necessário mudar o prefixo: 
```suggestion
export type TMediaPreviewProps = {
```